config.go

```diff
@@ -1055,3 +1057,37 @@ func (c ConfigTLS) LoadTLS(server string) (*tls.Config, error) {
 
 	return tlsConfig, nil
 }
+
+// Computes a hash of the cert files in the config
+func (c ConfigTLS) GetTLSHash() (string, error) {
```

Comment:
Also, why not compute the hash once, when the TLS file is loaded? Then store that has with the TLS config. (I've been away from this part of Blip code too long now, so I don't recall if this will work, but it seems like it should if the `ConfigTLS` struct is kept anywhere.)

Reply:
I can compute the hash when loading the TLS config, but I don't have a place to put it. `blip.TLSConfig` is passed as a struct for the `LoadTLS` function so I can't update it. I can return an additional value from that function that contains the hash. Really that saves reading the files twice but I'm not sure if it makes things too much cleaner.

Reply:
I've updated the code to try and blend the suggestions here. The hash is computed while loading the `tls.Config` but returned as another return value. This cuts down on calls a bit. I'm not putting it into the `Credentials` struct at this point to avoid loading the certs as part of `CredentialsFunc` running, though this certainly could be changed.